### PR TITLE
docs: Update Sendgrid Plugin Documentation

### PR DESCRIPTION
**Pull request by annalbirena**
Added a specific note regarding the use of localization keys in the context field of the cart, emphasizing the necessity of including these keys under `cart.context.locale`

### CODE REVIEW

File: www/apps/docs/content/plugins/notifications/sendgrid.mdx [hunk 1]
```diff
@@ -3953,7 +3953,7 @@ const plugins = [
 
 The `api_key` and `from` options are required. Then, use the key of each template you create (from the [reference](#template-reference)) as the option name with the template ID as the value.
 
-You can also optionally pass the option `localization` if you want to support different languages. `localization` accepts an object that has the country and language codes as keys. The value for each code should be an object of template keys and their IDs as values. Make sure to include the localized template IDs in `.env` as well.
+You can also optionally pass the option `localization` if you want to support different languages. `localization` accepts an object that has the country and language codes as keys. The value for each code should be an object of template keys and their IDs as values. It's important to note that the keys you add should also be present in the context field of the cart under the locale key `cart.context.locale`. This is crucial to ensure that the templates are used correctly based on the cart's localization. Make sure to include the localized template IDs in `.env` as well.
```
Comment (shahednasser) [inline www/apps/docs/content/plugins/notifications/sendgrid.mdx]:
```suggestion
You can also optionally pass the option `localization` if you want to support different languages. `localization` accepts an object that has the country and language codes as keys. The value for each code should be an object of template keys and their IDs as values. Make sure to include the localized template IDs in `.env` as well.

It's important to note that the keys you add should also be present in the context field of the cart under the locale key `cart.context.locale`. This is crucial to ensure that the templates are used correctly based on the cart's localization.
```

**Review by shahednasser — CHANGES_REQUESTED** [review]
Thanks for your contribution! Just have a small requested change.

**Comment by annalbirena** [comment]
It looks better now!

**Review by shahednasser — APPROVED** [review]
LGTM!

**Pull request merged**